src/pymovements/__init__.py

```diff
@@ -50,3 +51,8 @@
     'synthetic',
     'utils',
 ]
+
+__version__ = _version.get_versions()['version']
+
+from . import _version
```

Comment:
it seems you have forgotten to remove these two lines somehow?

Reply:
done -- must've been reattached during some kind of merging